app/team.go

```diff
@@ -1002,8 +1002,11 @@ func (a *App) LeaveTeam(team *model.Team, user *model.User, requestorId string)
 		}
 	}
 
-	err = a.RemoveTeamMemberFromTeam(teamMember, requestorId)
-	if err != nil {
+	if err = a.Srv().Store.Channel().UpdateSidebarChannelsCategoriesOnLeave(channel, team.Id); err != nil {
```

Comment:
Should this maybe be done in `RemoveTeamMemberFromTeam`? Is this called when the user is removed from a team instead of them just leaving a team?